app/controllers/application_controller.rb

```diff
@@ -3,6 +3,7 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include Pundit
```

Comment:
```suggestion
  include Pundit
  after_action :verify_authorized, except: :index
  after_action :verify_policy_scoped, only: :index
```

Reply:
Let's do that in the next PR - might be a good task for @JacOng17 to make sure everything is correctly authorized